content/docs/command-reference/add.md

```diff
@@ -243,3 +243,39 @@ In this case, a DVC-file is generated for each file in the `pics/` directory
 tree. No top-level DVC-file is generated, which is typically less convenient.
 For example, we cannot use the directory structure as one unit with `dvc run` or
 other commands.
+
+## Example: Dvcignore
+
+To untrack a file or directory just add
+[patterns](https://git-scm.com/docs/gitignore) associated with that file or
+directory under `.dvcignore` file.
+
+Let's take an example to illustrate this.
+
+```dvc
+$ mkdir dir
+$ echo file_one > dir/file1
+$ echo file_two > dir/file2
+```
+
+Now add `file1` to `.dvcignore` and track `dir` directory with `dvc add`.
+
+```dvc
+$ echo dir/file1 > .dvcignore
+$ dvc add dir
+$ cat .dvcignore
+dir/file1
+```
+
+Let's now modify the file which we added to `.dvcignore` and run `dvc status`.
+
+```dvc
+$ echo file_one_changed > dir/file1
+$ dvc status
+Data and pipelines are up to date.
+```
+
+No change has been detected by `dvc status` and hence we can say that it has
+untracked `file1` sucessfully.
```

Comment:
```suggestion
`dvc status` ignores changes to files listed in `.dvcignore`.
```

Reply:
Let's also add another part to explain this example after this paragraph showing that only `file2` is saved to the project cache please. The caching mechanism is explained in the `dvc add` cmd ref and the cache itself is explained in https://dvc.org/doc/user-guide/dvc-files-and-directories and in https://dvc.org/doc/command-reference/cache.